css/main.css

```diff
@@ -1,3 +1,733 @@
+  color: var(--red_brown);
+}
+
+.login,
+nav,
+.desktop-image-map {
+  display: none;
+}
+
+@media (min-width: 700px) {
+  .master-page {
+    margin: 0 20px;
+    display: grid;
+    grid-auto-rows: auto;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-areas:
+      "header header header header"
+      "main    main    main   main"
+      "footer  footer footer footer";
+  }
+
+  /* Header */
+
+  .header-site {
+    grid-area: header;
+  }
+
+  .search-container {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 40px;
+  }
+
+  .search-icon img {
+    left: 55px;
+  }
+
+  .login-bag {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+  }
+
+  .login {
+    margin-right: 50px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+  }
+
+  .login img {
+    width: 25px;
+    height: 25px;
+    filter: invert(43%) sepia(6%) saturate(4186%) hue-rotate(329deg)
+      brightness(89%) contrast(91%);
+  }
+
+  .login a {
+    color: var(--red_brown);
+    text-decoration: none;
+    margin-left: 15px;
+  }
+
+  .menu_burger,
+  .mobile-image-map {
+    display: none;
+  }
+
+  .header-logo_title_menu_burger {
+    margin: 0;
+    padding: 0;
+  }
+
+  nav {
+    display: block;
+  }
+
+  nav ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    padding-left: 7px;
+  }
+
+  nav li a {
+    text-decoration: none;
+    color: var(--red_brown);
+    padding-left: 20px;
+  }
+
+  .logo-nav {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .logo_title {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  /* Main */
+
+  .main {
+    grid-area: main;
+  }
+
+  .hero-image {
+    width: 100%;
+    /* height: 560px; */
+    background-image: url("../img/big-image.jpg");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .logo-icon-words {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .logo-icon img {
+    height: 6rem;
+    padding: 0 16px 0 14px;
+  }
+
+  .logo-words {
+    width: 3.4rem;
+  }
+
+  .logo-words p {
+    font-size: 0.7rem;
+  }
+
+  .hero-content h1 {
+    font-size: 4.7rem;
+  }
+
+  .hero-content h2 {
+    font-size: 1rem;
+    letter-spacing: 0.4rem;
+  }
+
+  .btn.shop-now {
+    font-size: 0.9rem;
+    padding: 0.6rem 3.2rem;
+  }
+
+  .feature-and-image {
+    /* width: 100%; */
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-around;
+  }
+
+  .feature-title-btn {
+    position: absolute;
+    width: auto;
+    padding: 14px 102px;
+    z-index: 1;
+  }
+
+  .image-products {
+    padding: 0 106px;
```

Comment:
This padding is causing trouble on desktop. You could achieve the desired effect more easily by using CSS Grid and setting a grid-gap?